cfme/fixtures/authentication.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+from time import sleep
+
+import cfme.utils.auth as authutil
+from cfme.utils.appliance import current_appliance
+from cfme.utils.log import logger
+
+
+def get_auth_settings(appliance):
+    """Grab the authentication settings from the UI form, popping the title widget content out"""
+    settings = {'auth_mode': appliance.server.authentication.auth_mode}
+    settings['auth_settings'] = appliance.server.authentication.auth_settings
+    settings['auth_settings'].pop('title')
```

Comment:
**QUESTION:** why pop the title?

Reply:
The auth_settings view will return the read from the title widget, which isn't actually part of the auth settings. This allows for taking the result from this and passing it back into `configure()` as a fill dict.

Reply:
**OPTIONAL:** Add comment with that explanation.

Reply:
See function docblock :) - want more than that?

Reply:
I'm actually going to collapse this into AuthenticationSetting.auth_settings property.

Reply:
So that docblock should not speak of implementation typically, and it seemed like what you were doing was an implementation thing to make passing it to configure possible.   Hence I would have probably expected that information to not be in the doc block but in a comment next to the implementation in question.   This is a tiny bit of code, so meh.